purl/purl.go

```diff
@@ -53,6 +53,8 @@ const (
 	TypeDebian = "deb"
 	// TypeDocker is a pkg:docker purl.
 	TypeDocker = "docker"
+	// TypeK8sDocker is a pkg:docker purl.
```

Comment:
Why docker?
Rename to just TypeK8s and the comment to "// TypeK8s is a pkg::k8s purl"